app/poros/board_game.rb

```diff
@@ -3,15 +3,17 @@
 class BoardGame
   attr_reader :name, :description, :duration, :image, :game_type
 
-  def initialize(data, categories = nil)
+  def initialize(data, categories)
     @name = data[:name]
     @description = data[:description]
     @min_age = data[:min_age]
     @duration = data[:max_playtime]
     @image = data[:image_url]
     @min_players = data[:min_players]
     @max_players = data[:max_players]
-    @game_type = categories
+    @available_categories = categories
+    @game_type_ids = data[:categories]
```

Comment:
Each game now knows of all categories, thoughts? Was not sure how else to translate ids to names without this list.